cmd/kfctl/cmd/replicate-build.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"fmt"
+	kftypes "github.com/kubeflow/kfctl/v3/pkg/apis/apps"
+	"github.com/kubeflow/kfctl/v3/pkg/kfapp/gcp"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var include string
+var exclude string
+var context string
+var outputFileName string
+
+func init() {
+	replicateBuildCmd.Flags().StringVarP(&include, "include", "i", "", "When set, only replicate images with matching prefix")
+	replicateBuildCmd.Flags().StringVarP(&exclude, "exclude", "e", "gcr.io", "Skip replicate images with matching prefix")
+	replicateBuildCmd.Flags().StringVarP(&context, "context", "c", "", "Path to build context, for example gs://...")
```

Comment:
Does this need to build a flag? Could a user just edit the PipelineRun that you output?